docs/modules/ROOT/pages/configuration/cache-expiration-time.adoc

```diff
@@ -0,0 +1,40 @@
+= Cache Expiration Time
+
+KubeArchive delegates authentication and authorization to Kubernetes. To
+avoid pressure on the Kubernetes API, KubeArchive caches the authentication
+and authorization answers. You can modify the expiration time of the cache.
+By default the expiration time is set to 10 minutes.
+
+. Prepare a patch for the `kubearchive-api-server` deployment. Set the value of the
+`AUTHENTICATION_CACHE_EXPIRATION` environment variable to a supported string (parseable by
+link:https://pkg.go.dev/time#ParseDuration[time.ParseDuration]), for example "1h":
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: Deployment
+spec:
+  template:
+    spec:
+      containers:
+      - name: kubearchive-api-server
+        env:
+        - name: AUTHENTICATION_CACHE_EXPIRATION
+          value: 1h
+----
+
+. Save the patch somewhere in the disk and apply it:
++
+[source,bash]
+----
+kubectl patch -n kubearchive deployment kubearchive-api-server --patch-file path/to/patch.yaml
+----
+
+[IMPORTANT]
+.Expiring the cache
+====
+To expire the current cache restart the `kubearchive-api-server` deployment with
+the command
+[command]`kubectl rollout -n kubearchive restart deployment kubearchive-api-server`.
+====
```

Comment:
There is a decision being made here. I don't have strong feelings against this and we could go with what you are proposing, but I would want to raise other options in here.

This PR is currently enforcing all authZ and authN calls to be cached (success and failures) and not providing a way to enforce token re-auth.
Pros:
* Harder to get slow responses due to the requests to the k8s API
Cons:
* If the permissions change, either forbidding access or allowing it we may need to wait the expiration caching time to be able to see the changes.

There is also the possibility of only caching success requests. Then if the client was unauthorized and ask for his permissions to be changed by a cluster admin, he can automatically see the changes there. The drawback here is that it potentially increases the slowdown of the API if there are too many failed unauthenticated / unauthorized requests.

Reply:
I see what you mean... we can also have different cache expiration for both but that would complicate the implementation... let me explore a little the implementations and then I will be back with a more complete answer.

Reply:
I introduced one variable to control the authorized/authenticated expiration and one to control the unauthorized/unauthenticated expiration. Its not that much complex in code and mentally, so I went for it.

Reply:
I like your approach. I made a couple of comments of where I would put the default values, but for the rest I am ok with the changes